### ISSUE TITLE
Request: add a specific Page to link to, about "Orders & Charter Groups"

### ISSUE THREAD

**Issue by olleolleolle** [drachenwald/drachenwald#128]
> The main link to Orders & Charter Groups redirect to
> https://drachenwald.sca.org/offices/
> instead of to a dedicated page for orders & charter groups.

Reported in https://github.com/drachenwald/dw_op/issues/69

**Comment by olleolleolle** [comment]
@dw-webminister Let's make an attempt to do that. But, it made me think, in a mobile view of the page, does that navigation even mean anything? Is that menu inside a hamburger menu, then? (I'm just thinking out loud here)

**Comment by dw-webminister** [comment]
yes - it's inside the 'toggle menu' button underneath the banner. (not a hamburger per se)
![image](https://github.com/drachenwald/drachenwald/assets/110694806/4cb91fe1-3fa0-473f-871d-b0ffbe037674)
